lib/processors/debugFileCreator.js

```diff
@@ -14,7 +14,7 @@ const util = require("util");
  */
 module.exports = function({resources, fs}) {
 	const options = {
-		pattern: /((\.view|\.fragment|\.controller|\.designtime|\.support)?\.js)/,
+		pattern: /((\.view|\.fragment|\.controller|\.designtime|\.support)?\.js)$/,
```

Comment:
```suggestion
		pattern: /((?:\.view|\.fragment|\.controller|\.designtime|\.support)?\.js)$/,
```

To align this fully with the Maven implementation (see `UI5NamingConventions.java`), we could also make the second group non-capturing. We don't reference it anyways.